flintrock/scripts/install-spark.sh

```diff
@@ -2,22 +2,20 @@
 
```

Comment:
Out of curiosity, why is the `download-hadoop` script written in python, whereas this one is in bash?